src/blockrelay/mempool_sync.h

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2018-2019 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_MEMPOOL_SYNC_H
+#define BITCOIN_MEMPOOL_SYNC_H
+
+#include "blockrelay/blockrelay_common.h"
+#include "blockrelay/graphene.h"
+#include "consensus/consensus.h"
+
+#include <chrono>
+
+const uint64_t MEMPOOL_SYNC_MIN_VERSION_SUPPORTED = 0;
+const uint64_t MEMPOOL_SYNC_MAX_VERSION_SUPPORTED = 0;
```

Comment:
please change these two constants to 
DEFAULT_MEMPOOL_SYNC_MIN_VERSION_SUPPORTED
DEFAULT_MEMPOOL_SYNC_MAX_VERSION_SUPPORTED
to match the naming convention of default values used by the rest of the codebase

Reply:
Done.